### ISSUE TITLE
Credential setup script

### ISSUE THREAD

**Issue by dominic22** [iotaledger/integration-services#360]
There is a lot of confusion how to start receiving a credential at the beginning. Since we follow the network of trust mechanism, we only are able to issue credential if the initiator is already trusted. But to have some initiator who is trusted we need to issue a credential to him. 

Therefor a local script should be used which can be run and:
1. Creates an identity
2. Sign a Credential to the identity
3. Stores the keys of the identity in a local file

The problem is described here:

![image.png](https://images.zenhubusercontent.com/60103daee540782abbba17ca/c50d54e7-aaa2-4fe5-9d79-eab67378a119)

**Comment by mastrogiovanni** [comment]
After creating the node SDK for integration services, we have now example-0 that does this work.
It is able to create an identity for an Administrator.
This script will require access to database and Integration Services.
 
https://github.com/iotaledger/integration-services/blob/develop/clients/node/examples/0-MakeRootIdentityAdmin.ts

**Comment by DocBO** [comment]
Hi, I tried to run the example-0 after resetting my database. What I encountered first, was that the API version of the example was set to v0.1, my API started with v1, so I changed that in the examples also to v1.
Second after checking the db url carefully and everthing I finally got this error: Error: Request failed with status code 400
And after some debugging I found this was caused by: createCredential call. Any idea?
